run-pr-comparisons

```diff
@@ -118,10 +118,12 @@ sed -i '/COMPARISON;/d' $WORKSPACE/results/testsResults.txt
 echo "COMPARISON;RUNNING" >> $WORKSPACE/results/testsResults.txt
 send_jenkins_artifacts $WORKSPACE/results/testsResults.txt ${PR_BASELINE_JOBDIR}/testsResults.txt
 
-if $DQM_COMPARISON_TEST ; then 
-  voms-proxy-init
-  python $CMS_BOT_DIR/dqm-comparison/compareDQMOutput.py -b ${JR_COMP_DIR}/$RELEASE_FORMAT/ -p ${JR_COMP_DIR}/PR-${PULL_REQUEST_NUMBER} -o $CMS_BOT_DIR/dqm-comparison/dqmComparisonOutput/ -n $PULL_REQUEST_NUMBER -r $RELEASE_FORMAT -s $WORKSPACE/upload/ -j $NUM_PROC 2> $WORKSPACE/upload/dqmBinByBinLog.log || true
-  echo "DQM_BIN_BY_BIN_COMPARISON;OK,DQM bin by bin comparison,See results,/SDT/jenkins-artifacts/${COMP_UPLOAD_DIR}/dqm-histo-comparison-summary.html" >> $WORKSPACE/results/testsResults.txt
+if $DQM_COMPARISON_TEST ; then
+  if command -v compareDQMOutput.py ; then
+    voms-proxy-init
```

Comment:
you remove this line. job launch script https://github.com/cms-sw/cms-bot/blob/master/docker_launcher.sh#L34 already run it.